### ISSUE TITLE
Problems in accessing to maps using Explorer 11.0.51

### ISSUE THREAD

**Issue by chiaracurcio** [geosolutions-it/MapStore2#2699]
### Description
Problems in accessing to maps using Explorer 11.0.51

### In case of Bug:
*Browser Affected*
- [x] Internet Explorer
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

*Browser Version Affected*
Explorer 11 (11.0.51)

*Steps to reproduce*
- access to the dev.mapstore2 using Explorer 11.0.51
- login 
- note that you can't see all maps

*Expected Result*
- You can see and open all the available maps

*Current Result*
- You can't see and open all the available maps

**Comment by offtherailz** [comment]
The problem is that internet explorer caches request's response (even if headers, in specific the authentication's one, has been changed). So if you change page of the map list, the new page will show the correct number of results. If you go back to the page that had problem, you will see again the problem.
I verified it is is caused by [Internet Explorer's ajax caching](http://www.dashbay.com/2011/05/internet-explorer-caches-ajax/). If you clear cache from developer tools the problem is solved.
We need to choose if we have to invalidate cache using query string parameters or provide 'correct' caching headers from GeoStore
